testutils/channel_opts.go

```diff
@@ -171,7 +172,11 @@ func DefaultOpts(opts *ChannelOpts) *ChannelOpts {
 
 // WrapLogger wraps the given logger with extra verification.
 func (v *LogVerification) WrapLogger(t testing.TB, l tchannel.Logger) tchannel.Logger {
+	counts := make([]*atomic.Uint32, len(v.Filters))
```

Comment:
shouldn't need pointers here. anything that's addressable can call pointer methods